docs/book/releases/v2.3.0.md

```diff
@@ -0,0 +1,84 @@
+<!-- markdownlint-disable MD034 -->
+# vSphere CSI Driver - v2.3.0 release
+
+## New Feature
```

Comment:
We added CSI and CNS metrics too. Any reason why that is not part of this release note?

Reply:
Metrics for Vanilla driver is part of 2.2 release too (https://github.com/kubernetes-sigs/vsphere-csi-driver/commit/048213304ad0abb9338dc9f1bc9e5eec9500d751) and (https://github.com/kubernetes-sigs/vsphere-csi-driver/commit/fb61db8ed20e769a34735bf71a635c04dc43c53f)

So ideally it should have been added to 2.2 release notes. Do you want me to add it there as part of this PR?

Reply:
As discussed, not adding this here